.travis.yml

```diff
@@ -9,7 +9,7 @@ env:
   - BUILD_ARCH=arm
 before_install: go get github.com/Masterminds/glide
 install: glide install
-script: GOARCH=$BUILD_ARCH GOOS=$BUILD_OS go build
+script: GOARCH=$BUILD_ARCH GOOS=$BUILD_OS go build -ldflags '-X main.version='$TRAVIS_TAG
```

Comment:
I tried `$TRAVIS_TAG` and `$(TRAVIS_TAG)` - both resulted in that actual string getting injected into the binary